### DISCUSSION TITLE
how to report runtime in log?

### DISCUSSION THREAD

**Discussion by inv1s10n** [The-OpenROAD-Project/OpenROAD#6127] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/d06eff2e-b947-412c-a297-781dceff101c)
I can get runtime by using orfs. How can I get such log when I run openroad and run cmd in cmdline directly?

**Comment by maliberty** [comment]
From the orfs Makefile:

export TIME_BIN   ?= env time
TIME_CMD = $(TIME_BIN) -f 'Elapsed time: %E[h:]min:sec. CPU time: user %U sys %S (%P). Peak memory: %MKB.'

**Reply by inv1s10n** [reply, marked as answer]
true.
hope can help others.
such cmd can report runtime.make sure that you have /usr/bin/time
env time  -f 'Elapsed time: %E[h:]min:sec. CPU time: user %U sys %S (%P). Peak memory: %MKB.' /usr/local/bin/openroad aes_nangate.tcl
